### PR TITLE
Portfolio-desktop-version

### PR DESCRIPTION
**Pull request by hamid-murambiwa**
This is a microverse project where I was required to create the desktop version sections for the Portfolio web page.
* Add index.html file
* Add *.css file 
* Add *.svg files

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @hamid-murambiwa,

**This is a great project you have going on, well-done lad**. 
   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: portfolio.css [hunk 1]
```diff
@@ -20,765 +90,601 @@
...
+  align-items: flex-end;
+  margin-top: 20vh;
+  width: 100%;
+}
 
-  .Nav-1 a {
-    color: #28352f;
-    font-size: 5.5vw;
-    padding-left: 5vw;
+@media (min-width: 991.99px) {
```
Comment (mke2111) [inline portfolio.css]: - [ ] The value for min-width property should be in range of 992px - 1100px. These are the accepted desktop screen breakpoints. Change your value. What you are using lies under the tablet screen sizes.

File: portfolio.css [hunk 2]
```diff
@@ -20,765 +90,601 @@
...
-    justify-content: space-around;
-    width: 100%;
-    margin-top: 3%;
-    margin-bottom: 3%;
-    background: #fff;
+  .social-icons {
+    width: 1.5vw;
+    height: 1.5vw;
+    margin-right: 0.1vw;
+    margin-left: 0.1vw;
```
Comment (mke2111) [inline portfolio.css]: OPTIONAL

- [ ] You have over 900 lines of CSS. Within these lines, so many properties have been repeated.
- Group similar properties used more than twice and then assign that class within the HTML structure
```
    display: flex;
    flex-direction: row;
    justify-content: space-around;
```
- Use the known CSS properties that combine more than one property.
```
margin: 0 0.1vw;
border: 1px solid red;
background: center cover no-repeat;
```

Adapt to these practices to keep your code DRY which is also the best practice of writing code.

**Review by george-shammar — APPROVED** [review]
Hi @hamid-murambiwa , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**